### DISCUSSION TITLE
Can i download video from udemy?

### DISCUSSION THREAD

**Discussion by huuluc10** [aandrew-me/ytDownloader#227] [Q&A] [answered]
I want to download video from course on udemy which I bought. I try download but it's forbidden. Can i add authorization token into download request?

ERROR: [udemy:course] course: Unable to download webpage: HTTP Error 403: Forbidden (caused by )

**Comment by aandrew-me** [comment, marked as answer]
In preferences there is an option to use cookie from a browser. Select the browser where you're logged in to Udemy

**Reply by huuluc10** [reply]
Thanks for the answer, but it seems the videos on udemy are DRM protected so I can't download them.

**Reply by aandrew-me** [reply]
Well that's disappointing
